zkevm-circuits/src/pi_circuit.rs

```diff
@@ -95,6 +95,16 @@ pub struct PublicData {
     pub max_calldata: usize,
     /// Max number of supported inner blocks in a chunk
     pub max_inner_blocks: usize,
+    /// Previous last applied l1 block
+    pub prev_last_applied_l1_block: u64,
+    /// Last applied l1 block
+    pub last_applied_l1_block: u64,
+    /// L1 block range hash
+    pub l1_block_range_hash: Hash,
```

Comment:
similar as the previous comment. This can be removed and recomputed using the `cum_l1_block_hashes` field.